### PR TITLE
CAKE-5019 | Add all tracking events

### PR DESCRIPTION
**Pull request by vforge**

### CODE REVIEW

File: src/Tracker.js [hunk 1]
```diff
@@ -12,6 +12,10 @@ const TRACK_PARAMS = {
 };
 const TRACK_TIMEOUT = 3000;
 
+const TRACKING_CATEGORY = 'gdpr-modal';
+const ACTION_IMPRESSION = 'Impression';
```
Comment (mklucsarits) [inline src/Tracker.js]: Are you sure these should be capitalized? They're lowercase in the spreadsheet.

File: src/Tracker.js [hunk 2]
```diff
@@ -77,6 +83,96 @@ class Tracker {
...
+    /**
+     * All tracking events
+     * @see https://docs.google.com/spreadsheets/d/1SdKVYFpAcW4xnLDOhLmpZ1h-Kr8Dq2a0tk-fb_GEEms/edit#gid=0
+     */
+    trackViewImpression() {
+        this.trackImpression('modal-view');
+    }
+
+    trackNoCookieImpression() {
+        this.trackImpression('no-coolor');
```
Comment (mklucsarits) [inline src/Tracker.js]: Typo (this event happens when you forget to pack a cooler for your camping trip).

File: src/Tracker.js (hunk 1 above)
Comment (vforge) [reply src/Tracker.js]: That's the original piece of the code https://github.com/Wikia/tracking-opt-in/pull/81/files#diff-57aa11691094c3295e1f9dcbe8432ba2L8

But you're right, I'll downcase them and check with Keiko.

File: src/Tracker.js (hunk 2 above)
Comment (vforge) [reply src/Tracker.js]: omg :shy:

File: src/Tracker.js (hunk 1 above)
Comment (mklucsarits) [reply src/Tracker.js]: ok I see that's how they were originally so 🤷‍♂

Comment (vforge) [reply src/Tracker.js]: They're lowercased in other products... 🤷‍♂

**Pull request merged**